spec/ffprober/ffprobe_version_spec.rb

```diff
@@ -15,7 +15,8 @@
     { version: '2.1.2', pass: true },
     { version: '2.1.4', pass: true },
     { version: '2.2',   pass: true },
-    { version: '2.2.2', pass: true }
+    { version: '2.2.2', pass: true },
+    { version: '2.5.4', pass: true }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.